uhubctl.c

```diff
@@ -19,7 +19,11 @@
 #include <ctype.h>
 #include <unistd.h>
 
+#ifdef __FreeBSD__
+#include <libusb.h>
```

Comment:
Hmm. I don't mind it, but surprised that FreeBSD does not have it under `<libusb-1.0/libusb.h>`?